docs/source/onnxruntime/modeling_ort.mdx

```diff
@@ -12,13 +12,13 @@ specific language governing permissions and limitations under the License.
 
 # Optimum Inference with ONNX Runtime
 
-Optimum is a utility package for building and running inference with accelerated runtime like ONNX Runtime. 
-Optimum can be used to load optimized models from the [Hugging Face Hub](hf.co/models) and create pipelines 
+Optimum is a utility package for building and running inference with accelerated runtime like ONNX Runtime.
+Optimum can be used to load optimized models from the [Hugging Face Hub](hf.co/models) and create pipelines
```

Comment:
are you using some kind of lint/prettier? (Just asking)

Reply:
I switched on the setting in VSCode to trim extra whitespaces. But I can revert this if you think it should not be done in this PR.

Reply:
Hehe, I asked the same question before.